doc/themes/the-things-stack/layouts/partials/proto/glossary-term.html

```diff
@@ -0,0 +1,18 @@
+<dl class='glossary'>
```

Comment:
I don't think this is correct use of the `dl` tag, since we're only outputting a single item here. Shouldn't this partial only output the `dt` and `dd`?

Reply:
It's more logical, yep, but outputting multiple `dl` gives flexibility to divide the glossary in to sections, e.g by letter - copied from [here](http://html5doctor.com/element-index/)

Pierre mocked it up that way and it could be nice to add if someone has the time. Multiple `dl` each with a single definition looks the same as a single `dl` with multiple definitions if there's no extra padding or margin.

![Image d’iOS](https://user-images.githubusercontent.com/6963436/80577444-2d70e900-8a07-11ea-9f43-22ec78bb1ad9.jpg)